app/Http/Controllers/ArticleCategoryController.php

```diff
@@ -12,54 +17,58 @@ class ArticleCategoryController extends Controller
      */
     public function index()
     {
-        //
+        $query=ArticleCategory::Query();
```

Comment:
To make your code a little neat, follow PSR conventions.
for example
```php
$query=ArticleCategory::Query();
```
when i format your document 
```php
$query = ArticleCategory::Query();
```

but it is a little concern, just use a good formatter
if you are using PHPStorm press `Ctrl` + `Alt` + `L`
if you are using VS code download extension called phpfmt and press `Ctrl` + `Shift` + `I` when you want to format the document